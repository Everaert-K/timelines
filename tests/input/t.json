[
    ["cow","chicken","cat", "scream", "dog"],
    [ "scream", "rrrr"]
]