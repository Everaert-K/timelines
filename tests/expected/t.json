[
["cow","chicken","cat","scream","dog"],
["cow","chicken","cat","scream","rrrr"]
]
